pages/account/addressBook.tsx

```diff
@@ -0,0 +1,24 @@
+import BaseTemplate from "@/components/BaseTemplate";
+import { NavigationPanel } from "@/components/NavigationPanel";
+
+const AddressBookPage: React.VFC = () => {
+  return (
+    <BaseTemplate>
```

Comment:
I would like to have `AccountBaseTemplate` component, so this page would become:

```
<AccountBaseTemplate>
<div>Address Book Component goes here</div>
</AccountBaseTemplate>
```

Account pages have shared navigation so it would be a good idea to not reimplement layout at every page